utils.js

```diff
@@ -0,0 +1,13 @@
+function shouldUseInstrumentedBuild() {
+  var args = process.argv;
+  for (var i = 0; i < args.length; i++) {
+    if (args[i] === '--instrument') {
```

Comment:
@stefanpenner should we have a convention for this in ember-cli? Also, if .debug vs .prod were made library convention, seems like this would conflict. Do we need instrument turned on/off for specific libraries?   We could use a define const and dead code elimination to do this in ember-cli.